templates/Makefile.build-harness-openshift-ci

```diff
@@ -1,6 +1,6 @@
 export SHELL = /bin/bash
-export GITHUB_USER ?=
-export GITHUB_TOKEN ?=
+export GITHUB_USER = acm-cicd%40redhat.com
+export GITHUB_TOKEN := $(shell cat /etc/github/token 2> /dev/null )
```

Comment:
Wouldn't these override the default's for all users?

Reply:
That makefile template is only used on prow. Repos using travis use the Makefile.build-harness-vendorized template

Reply:
Hmm - I know we've had issues with `make` preemptively running these statements even though they weren't in execution path - one way to check would be to run a command from a different module that requires these vars to have another value - I'm working on my own fork of bhe right now so it would be a bit of a horror to switch over - @Kyl-Bempah could you test right quick?

Reply:
This particular change is in a template, not a module. If you don't include this file, make isn't going to do anything with it.

Reply:
I'll approve it, but I assumed our make included anything with Makefile* in it.